tests/benchmark/benchmark.py

```diff
@@ -289,6 +289,7 @@ def run_test(setup):
         if state_override:
             log_prefix = \
                 f'WARNING: Previous test results does not have `.execution.results.json` file, so comparision with None was performed instead!\n' \
+                f'WARNING: This can often be fixed by merging with the latest `main`.\n' \
```

Comment:
@roccomoretti do we need dot at the end of this line? 🤔

Reply:
The period at the end of the sentence should work.